CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.0 [BREAKING]
```

Comment:
Wouldn't it be better if we add a section titled `breaking` or `breaking change` as a separate section under the version number instead of adding it on the side?